src/lib/device/virtual_dev.cpp

```diff
@@ -68,7 +69,7 @@ static std::shared_ptr< BlkAllocator > create_blk_allocator(blk_allocator_type_t
                                   size,
                                   is_auto_recovery,
                                   std::string("varsize_chunk_") + std::to_string(unique_id),
-                                  is_data_drive_hdd() ? false : true /* use_slabs */};
```

Comment:
why this has anything to do with `is_data_drive_hdd`?

Reply:
for non-metablk use case, which is the user data blk allocation, when it is HDD drives (for nublox use case), we adapt to HDD streams for best performance, which is each *user data* blk allocation inside a volume is directed to a stream (unquely owned by this volume), so that random write on this volume becomes sequencial write on this stream (internally mapped to a set of physical chunks). If use_slabs is true, stream concept will be overriden and it will choose blks from slabs which is not preferred in HDD use case.